<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotionManagerguiClass</class>
 <widget class="QWidget" name="MotionManagerguiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MotionManagergui</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QGroupBox" name="gb_MRelative">
       <property name="title">
        <string>单轴相对运动</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pb_AxisStop1">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_XRelativeValue">
            <property name="placeholderText">
             <string>X轴相对位移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_XRelativeGo">
            <property name="text">
             <string>启动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pb_AxisStop2">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_YRelativeValue">
            <property name="placeholderText">
             <string>Y轴相对位移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_YRelativeGo">
            <property name="text">
             <string>启动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pb_AxisStop3">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_ZRelativeValue">
            <property name="placeholderText">
             <string>Z轴相对位移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_ZRelativeGo">
            <property name="text">
             <string>启动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_MAbsolute">
       <property name="title">
        <string>单轴绝对运动</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pb_AxisStopAbso1">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_XAbsolute">
            <property name="placeholderText">
             <string>X轴绝对位移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_XAbsoluteGo">
            <property name="text">
             <string>启动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pb_AxisStopAbso2">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_YAbsolute">
            <property name="placeholderText">
             <string>Y轴绝对位移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_YAbsoluteGo">
            <property name="text">
             <string>启动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pb_AxisStopAbso3">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_ZAbsolute">
            <property name="placeholderText">
             <string>Z轴绝对位移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_ZAbsoluteGo">
            <property name="text">
             <string>启动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <widget class="QGroupBox" name="gb_ActPos">
       <property name="title">
        <string>轴实时位置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>X轴：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_XActPos"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Y轴：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_YActPos"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Z轴：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_ZActPos"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_LimInfo">
       <property name="title">
        <string>限位信息</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>X轴限位信息：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_XLimtN">
            <property name="text">
             <string>负限位</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/seagreen.png</normaloff>images/seagreen.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_XORG">
            <property name="text">
             <string>原点</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/seagreen.png</normaloff>images/seagreen.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_XLimtP">
            <property name="text">
             <string>正限位</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/seagreen.png</normaloff>images/seagreen.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Y轴限位信息：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_YLimtN">
            <property name="text">
             <string>负限位</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/seagreen.png</normaloff>images/seagreen.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_YORG">
            <property name="text">
             <string>原点</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/seagreen.png</normaloff>images/seagreen.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_YLimtP">
            <property name="text">
             <string>正限位</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/seagreen.png</normaloff>images/seagreen.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Z轴限位信息：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_ZLimtN">
            <property name="text">
             <string>负限位</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/seagreen.png</normaloff>images/seagreen.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_ZORG">
            <property name="text">
             <string>原点</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/seagreen.png</normaloff>images/seagreen.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_ZLimtP">
            <property name="text">
             <string>正限位</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/seagreen.png</normaloff>images/seagreen.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>三轴联动</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QPushButton" name="pb_3AxisStop">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_3AxisXpos">
          <property name="placeholderText">
           <string>X轴坐标</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_3AxisYpos">
          <property name="placeholderText">
           <string>Y轴坐标</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_3AxisZpos">
          <property name="placeholderText">
           <string>Z轴坐标</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_3AxisGo">
          <property name="text">
           <string>启动</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>输出端口</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QPushButton" name="pb_output1">
          <property name="text">
           <string>输出1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output2">
          <property name="text">
           <string>输出2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output3">
          <property name="text">
           <string>输出3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output4">
          <property name="text">
           <string>输出4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output5">
          <property name="text">
           <string>输出5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output6">
          <property name="text">
           <string>输出6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output7">
          <property name="text">
           <string>输出7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output8">
          <property name="text">
           <string>输出8</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="pb_output9">
          <property name="text">
           <string>输出9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output10">
          <property name="text">
           <string>输出10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output11">
          <property name="text">
           <string>输出11</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output12">
          <property name="text">
           <string>输出12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output13">
          <property name="text">
           <string>输出13</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output14">
          <property name="text">
           <string>输出14</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output15">
          <property name="text">
           <string>输出15</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_output16">
          <property name="text">
           <string>输出16</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>输入端口</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QPushButton" name="pb_input1">
          <property name="text">
           <string>输入1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input2">
          <property name="text">
           <string>输入2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input3">
          <property name="text">
           <string>输入3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input4">
          <property name="text">
           <string>输入4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input5">
          <property name="text">
           <string>输入5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input6">
          <property name="text">
           <string>输入6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input7">
          <property name="text">
           <string>输入7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input8">
          <property name="text">
           <string>输入8</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QPushButton" name="pb_input9">
          <property name="text">
           <string>输入9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input10">
          <property name="text">
           <string>输入10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input11">
          <property name="text">
           <string>输入11</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input12">
          <property name="text">
           <string>输入12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input13">
          <property name="text">
           <string>输入13</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input14">
          <property name="text">
           <string>输入14</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input15">
          <property name="text">
           <string>输入15</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_input16">
          <property name="text">
           <string>输入16</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QTextEdit" name="te_InitStatus"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MotionManagergui.qrc"/>
 </resources>
 <connections/>
</ui>
